infra/app/database.bicep

```diff
@@ -60,6 +63,11 @@ var containers = [
           type: 'diskANN'
         }
       ]
+      fullTextIndexes: [
+        {
+          path: '/description',
```

Comment:
Weren't we going to add FTS search to the tags too?

Reply:
Updated to tags now.